### ISSUE TITLE
Assiged Characters are not showing up to be picked  -- CANCEL plz -- 

### ISSUE THREAD

**Issue by jseaz73** [League-of-Foundry-Developers/fvtt-module-lmrtfy#148]
When I open LMRFY no mobs show up ... 

and when I toggle to Assigned Chars ...

I get this error in the logs 
backend.mjs:306 Foundry VTT | Created Token with id [VHNzWTbwZ4TLUVnz] in parent Scene [H4e97sVZaPtWdMSX]
foundry.js:3053 Foundry VTT | Rendering TokenMoldOverlay
3foundry.js:3053 Foundry VTT | Rendering TokenMoldOverlay
requestor.js:131 Uncaught TypeError: Cannot read properties of undefined (reading 'map')
[Detected 1 package: lmrtfy]
    at LMRTFYRequestor._getUserActorIds (requestor.js:131:42)
    at LMRTFYRequestor._onUserChange (requestor.js:143:29)
    at HTMLSelectElement.dispatch (jquery.min.js:2:43064)
    at HTMLSelectElement.v.handle (jquery.min.js:2:41048)
_getUserActorIds @ requestor.js:131
_onUserChange @ requestor.js:143
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2

}} The updater did not see it was NOT uptodate ... and so I had to remove it and reinstall 2.0 ...

FIXED {{

**Comment by Varriount** [comment]
You know you can close issues on your own, right?

**Comment by jseaz73** [comment]
I apologize.  I looked for it but I did not see it :-(  

At least I fixed the problem on my own.   It seems that 9.249 did not see this as NOT being updated along with a couple of others too... I had to remove the modules completely and then add them again to enable them to work.

It is as if the upgrade did not update the stats when I went into FVTT and opened the Manage Modules --- the modules showed as NOT being compliant with 9.249  but the Module Updater showed them as up 2 date.
